widgets/common/src/filters/consts.ts

```diff
@@ -1,3 +1,3 @@
-// eslint-disable-next-line import/prefer-default-export
 export const { i18n } = Stage;
 export const i18nPrefix = 'widgets.common.filters.form';
+export const i18nPlaceholdersPrefix = 'widgets.common.filters.form.inputsPlaceholders';
```

Comment:
Optional:

We could reference the `i18nPrefix` here

```suggestion
export const i18nPlaceholdersPrefix = `${i18nPrefix}.inputsPlaceholders`;
```

Reply:
Right, fixed.